File: JS/Account.js
class Account {
    constructor(name,document, email, password){
        this.id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;

    }
     printUser(){
         console.log("the user is: "+this.name);
         console.log("the document is: "+this.document);
         console.log("the email is: "+this.email);
         console.log("the password is: "+this.password);
     }
}

File: java/Account.java
class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name, String document){
        this.name = name;
        this.document = document;
    }

  
    void printUser(){
      System.out.println("User: "+name+" document: "+document+" email: "+email+" password: "+password);  
  }
}